gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/transports/base.py.j2

```diff
@@ -18,9 +18,14 @@ from google.auth import credentials as ga_credentials  # type: ignore
 from google.oauth2 import service_account # type: ignore
 
 {% filter sort_lines %}
+{% set import_ns = namespace(has_operations_mixin=false) %}
```

Comment:
It seems like this may need to be generalized to the other mixins. WDYT?

I don't think we should block on that, but I would add a TODO in the code and reference this PR.